slides/06_Hooks.html

```diff
@@ -50,12 +50,17 @@ <h3>The <span class="warn">useState</span> Hook</h3>
               function MyComponent() {
                 // Define a name state with the initial value of ""
                 const [name, setName] = useState("");
-            
-                // The setName function updates the state
-                setName("Joe");
-
+              
+                // setName updates the state
+                const updateName = (e) => setName(e.target.value);
```

Comment:
The idea of this page is to explaine what the two array elements returned by useState are. Felix suggested to instead of throw it inside a component, to use multiple code snippets, so it's clear that they are not executed as seen above. With the change proposed by you we take already over things from the upcoming pages.
I'd go for following four separate code snippets (where number 3 was missing, but in some cases is very handy):
```
// Define a name state with the initial value of ""
                const [name, setName] = useState("");
```

```
// The setName function updates the state
                setName("Joe");
```

```
// Alter the existing Name
                setName(oldName => `Mr. ${oldName}`);
```

```
// The variable 'name' contains the current state value
                return <p>Name: {name}</p>
```

Reply:
That's a good point I didn't consider. I'll split it up into multiple code snippets.